eea/facetednavigation/widgets/checkbox/widget.py

```diff
@@ -36,8 +36,9 @@ def css_class(self):
         """
         css_type = self.widget_type
         css_title = normalizer.normalize(self.data.title)
+        custom_css = ' {}'.format(self.data.custom_css) if len(self.data.custom_css) else ''
```

Comment:
Hmm, I wonder if this is backward compatible. In the sense that you already have a checkbox widget within your site. Do an upgrade to eea.facetednavigation :thinking: 

It may raise AttributeError: `custom_css`

I'll give it a try.

Reply:
we may have to replace it with:
```
getattr(self.data, 'custom_css', '')
```

or

```
self.data.get('custom_css', '')
```

Reply:
Oh... I see what you mean. This makes sense. I think @Gomez stumbled into something like that ... in my testing, I was not able to reproduce it.

Reply:
I will update that. Thx.

Reply:
For some reasons it didn't fail as I expected, but still 
```
self.data.get('custom_css', '')
```

would be a better approach, just to stay on the safe side :wink: